modules/admin_manual/pages/configuration/user/oidc/index.adoc

```diff
@@ -6,53 +6,236 @@
+
+TIP: Kopano Konnect can be set up via Docker. Images are available on Docker Hub (`kopano/kopano_konnect`).
+TIP: For general information on Kopano Konnect, please refer to the https://documentation.kopano.io/kopanocore_administrator_manual/configure_kc_components.html#configure-kopano-konnect[Kopano Documentation]. The configuration below is an example to make Kopano Konnect work with ownCloud Server.
+
+==== Basic configuration
+
+To run Kopano Konnect, some basic configuration is necessary. Please replace the below environment variables with the paths and values suited for your environment.
+
+```
+- FQDN=idp.example.com
+- allow_client_guests=yes
+- allow_dynamic_client_registration=yes
+- eckey=/kopano/ssl/meet-kwmserver.pem
+- ecparam=/kopano/ssl/ecparam.pem
+- encryption_secret_key=/kopano/ssl/konnectd-encryption.key
+- identifier_registration_conf=/kopano/ssl/konnectd-identifier-registration.yaml
+- identifier_scopes_conf=/etc/kopano/konnectd-identifier-scopes.yaml
+- signing_private_key=/kopano/ssl/konnectd-tokens-signing-key.pem
+- validation_keys_path=/kopano/ssl/konnectkeys
+```
+
+==== Connect Kopano Konnect to LDAP/AD
+
+To allow Kopano Konnect to authenticate ownCloud users it is necessary to connect it to the same LDAP/AD server which is used by the ownCloud Server. Below you find the environment variables needed to connect Kopano Konnect to an LDAP/AD server. Please replace them with the values suited for your environment (e.g., from the ownCloud LDAP configuration).
+
+```
+- KONNECT_BACKEND=ldap
+- LDAP_BASEDN=dc=owncloud,dc=com
+- LDAP_BINDDN=cn=admin,dc=owncloud,dc=com
+- LDAP_BINDPW=password
+- LDAP_EMAIL_ATTRIBUTE=mail
+- LDAP_FILTER=(objectClass=organizationalPerson)
+- LDAP_LOGIN_ATTRIBUTE=uid
+- LDAP_NAME_ATTRIBUTE=cn
+- LDAP_SCOPE=sub
+- LDAP_URI=ldap://<ldap-address>:389
+- LDAP_UUID_ATTRIBUTE_TYPE=text
+- LDAP_UUID_ATTRIBUTE=uidNumber
+```
+
+==== Register ownCloud Clients
+
+To allow the ownCloud Clients (Web/Desktop/Android/iOS) to interact with Kopano Konnect, you have to register them as clients using Konnect's `identifier-registration.yaml`.
+Below you find an exemplary `identifier-registration.yaml`. It is based on the default values for the regular ownCloud Clients and works with the configuration of ownCloud Server provided in the sections below. Other environments might require a different set of values. 
+
+TIP: When registering ownCloud as and OpenID Client, use `https://cloud.example.net/index.php/apps/openidconnect/redirect` as the redirect URL. In case {openid-connect-frontchannel-logout-url}[OpenID Connect Front-Channel Logout 1.0] is supported, please enter `https://cloud.example.net/index.php/apps/openidconnect/logout` as the logout URL within the client registration of the OpenID Provider.
```

Comment:
afaik kopano is not supporting front-channel-logout (at least it did not months ago) so it is pointless to talk about it here.
please verify with upstream